### PR TITLE
Add admin page

### PR DESCRIPTION
**Pull request by liubaterehova**

### CODE REVIEW

File: src/pages/admin/admin.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.row {
+  margin: 0;
+}
```
Comment (viento006) [inline src/pages/admin/admin.scss]: leave blank line at the end of file

Comment (viento006) [reply src/pages/admin/admin.scss]: You can add these settings to vscode to make life easier:
    "files.trimTrailingWhitespace": true,
    "files.insertFinalNewline": true,
    "files.trimFinalNewlines": true,

File: src/components/cards/card-admin/card-admin.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+import { PropTypes } from 'prop-types';
+
+import {
+  Col, Card, CardTitle, Button,
+} from 'reactstrap';
+
+import { useModals } from '../../../hooks/use-modals';
+
+export const CardAdmin = ({
```
Comment (viento006) [inline src/components/cards/card-admin/card-admin.jsx]: AdminCard

File: src/components/cards/card-film/card-film.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import { PropTypes } from 'prop-types';
+import {
+  Card, CardImg, CardText, CardBody,
+  CardTitle, Button,
+} from 'reactstrap';
+
+export const CardFilm = ({
```
Comment (viento006) [inline src/components/cards/card-film/card-film.jsx]: FilmCard

File: src/components/cards/cards-add-inf/cards-add-inf.jsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import { ModalAddCinema } from '../../modals/modal-add-cinema';
```
Comment (viento006) [inline src/components/cards/cards-add-inf/cards-add-inf.jsx]: rename file and folder. what is inf?

File: src/components/footer/footer.scss [hunk 5]
```diff
@@ -1,3 +1,11 @@
-
-
-
+.text-footer {
+  position: relative;
+  &:before {
```
Comment (viento006) [inline src/components/footer/footer.scss:4]: add an empty line before new style

File: src/components/forms/form-add-cinema.jsx [hunk 6]
```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { Form, Input, Col } from 'reactstrap';
+import PropTypes from 'prop-types';
+
+import { FormElement } from './form-element';
+
+const formElementAddCinema = [
```
Comment (viento006) [inline src/components/forms/form-add-cinema.jsx]: rename it

File: src/components/forms/form-add-cinema.jsx [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+  {
+    id: 5,
+    as: Input,
+    type: 'checkbox',
+    name: 'drinks',
+    label: 'Drinks',
+  },
+];
+
+export const FormAddCinema = ({
```
Comment (viento006) [inline src/components/forms/form-add-cinema.jsx]: AddCinemaForm

File: src/components/forms/form-add-film.jsx [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+  {
+    id: 5,
+    as: Input,
+    type: 'file',
+    name: 'file',
+    label: 'File',
+  },
+];
+
+export const FormAddFilm = ({
```
Comment (viento006) [inline src/components/forms/form-add-film.jsx]: AddFilmForm

Comment (viento006) [reply src/components/forms/form-add-film.jsx]: or CreateFilmForm

File: src/components/forms/form-add-film.jsx [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+  handleSubmit, errors, control,
+}) => {
+  const onSubmit = () => {};
+
+  return (
+    <Form onSubmit={handleSubmit(onSubmit)}>
```
Comment (viento006) [inline src/components/forms/form-add-film.jsx]: Why do you need this component at all? it's completely the same as AddCinema

File: src/components/forms/form-add-session.jsx [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+    as: DollarInput,
+    type: 'number',
+    name: 'price',
+    label: 'Price',
+    rules: { required: true },
+    errorMessage: 'This field is required',
+  },
+];
+
+export const FormAddSession = ({
```
Comment (viento006) [inline src/components/forms/form-add-session.jsx]: AddSessionForm

File: src/components/forms/form-sign-up.jsx [hunk 11]
```diff
@@ -63,6 +63,7 @@ export const FormSignUp = ({
         id, name, label, placeholder, type, errorMessage, rulesValidation,
```
Comment (viento006) [inline src/components/forms/form-sign-up.jsx:64]: SignUpForm

File: src/components/forms/form-sign-in.jsx [hunk 12]
```diff
@@ -33,6 +33,7 @@ export const FormSignIn = ({
       id, name, label, placeholder, type, rulesValidation, errorMessage,
     }) => (
       <FormElement
+        as={Input}
```
Comment (viento006) [inline src/components/forms/form-sign-in.jsx:36]: SignInForm

File: src/components/modals/modal-add-cinema.jsx [hunk 13]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useForm } from 'react-hook-form';
+
+import { ModalWindow } from './modals';
+import { FormAddCinema } from '../forms/form-add-cinema';
+
+export const ModalAddCinema = ({
```
Comment (viento006) [inline src/components/modals/modal-add-cinema.jsx]: AddCinemaModal

File: src/components/modals/modal-add-film.jsx [hunk 14]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useForm } from 'react-hook-form';
+
+import { ModalWindow } from './modals';
+import { FormAddFilm } from '../forms/form-add-film';
+
+export const ModalAddFilm = ({
```
Comment (viento006) [inline src/components/modals/modal-add-film.jsx]: AddFilmModal

**Review by viento006 — CHANGES_REQUESTED** [review]

File: src/components/forms/form-add-film.jsx (hunk 9 above)
Comment (liubaterehova) [reply src/components/forms/form-add-film.jsx]: I made the same components cause when I made sign in and sign up forms Ilia told me to do two forms and two modals and I made this forms and modals similarly

**Review by viento006 — APPROVED** [review]

**Pull request merged**